CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# Changelog
+
+All breaking changes to this project will be documented in this file, most recent changes at the top.
+
+## 0.5.0
+
+In an effort to clean up the previous change and improve it, we re-modified the dynamic-delay and time-tracking filters to extricate their configuration entirely from nighthawk.server.ResponseOptions.
```

Comment:
Can we enhance this paragraph with some permalinks to the proto definitions?

Ideally we will have:
- a link to the own configuration protos the dynamic-delay and time-tracking filters now have.
- convert nighthawk.server.ResponseOptions to a permalink.

Reply:
Created links for all of the proto configurations